### PR TITLE
Color contrast in blockquotes

### PR DESCRIPTION
**Pull request by AlexBrizi**
### Summary
Updated the color of text when using blockquotes to adhere to accessible contrast ratios of text to background color.

Resolves [#2188](https://github.com/cerner/terra-core/issues/2188)

### CODE REVIEW

File: scripts/a11y/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+const fs = require('fs');
```
Comment (emilyrohrbough) [inline scripts/a11y/index.js]: Do we to add this script in terra-core? Should we be checking for accessibility with webdriver.io tests for the markdown component?

File: packages/terra-markdown/src/Markdown.scss [hunk 2]
```diff
@@ -5,3 +5,7 @@
     background-color: #ebebef;
   }
 }
+
+.markdown-body blockquote {
```
Comment (emilyrohrbough) [inline packages/terra-markdown/src/Markdown.scss]: Can we add an webdriver.io accessibility test for this change?

File: scripts/a11y/index.js (hunk 1 above)
Comment (bjankord) [reply scripts/a11y/index.js]: This file shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.

File: scripts/a11y/a11y-output.txt [hunk 3]
```diff
@@ -0,0 +1,1346 @@
+[ { violations: [],
```
Comment (bjankord) [inline scripts/a11y/a11y-output.txt]: This file shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.

File: package.json [hunk 4]
```diff
@@ -35,12 +35,14 @@
     }
   },
   "scripts": {
+    "a11y": "node scripts/a11y/index.js",
```
Comment (bjankord) [inline package.json]: This file shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.

File: package.json [hunk 5]
```diff
@@ -35,12 +35,14 @@
...
     "bootstrap": "lerna bootstrap",
     "bootstrap:hoist": "lerna bootstrap --hoist --nohoist=moment --nohoist=one-cerner-style-icons",
     "clean:all": "lerna clean --yes && node scripts/clean-modules/index.js",
     "clean:install": "npm run clean:all && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
     "compile": "lerna run compile",
+    "compiledRouteConfig": "babel dev-site-config/build/routeConfig.js --out-file dev-site-config/build/compiledRouteConfig.js",
```
Comment (bjankord) [inline package.json]: This change shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.

File: package.json [hunk 6]
```diff
@@ -67,6 +69,8 @@
     "wdio": "cross-env NODE_OPTIONS=--max-old-space-size=2048 wdio node_modules/terra-dev-site/config/wdio/wdio.conf.js"
   },
   "devDependencies": {
+    "@babel/register": "^7.0.0",
```
Comment (bjankord) [inline package.json]: This change shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.

File: package.json [hunk 7]
```diff
@@ -67,6 +69,8 @@
...
+    "axe-puppeteer": "^0.1.0",
```
Comment (bjankord) [inline package.json]: This change shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.

File: package.json [hunk 8]
```diff
@@ -89,6 +93,7 @@
     "lerna": "^2.1.2",
     "link-parent-bin": "^0.2.0",
     "markdown-magic": "^0.1.25",
+    "puppeteer": "^1.11.0",
```
Comment (bjankord) [inline package.json]: This change shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.

File: package.json [hunk 9]
```diff
@@ -100,5 +105,8 @@
     "terra-dev-site": "^2.0.0",
     "terra-toolkit": "^4.5.0",
     "webpack-merge": "^4.1.2"
+  },
+  "dependencies": {
```
Comment (bjankord) [inline package.json]: This change shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.
